app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
```

Comment:
Consider closing the database connection when it's no longer needed to avoid resource leaks. You can implement a method to close the connection or use context managers.